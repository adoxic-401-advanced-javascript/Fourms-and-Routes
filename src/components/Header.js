import React from 'react';

const Headers = () => {
  return (
    <header>
      <h1>Aliens?????</h1>
    </header>
  );
};

export default Headers;
