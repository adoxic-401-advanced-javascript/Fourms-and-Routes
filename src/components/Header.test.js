import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header Test', () => {
  it('Renders an Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
