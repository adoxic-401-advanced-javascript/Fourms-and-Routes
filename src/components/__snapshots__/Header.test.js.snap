// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Test Renders an Header 1`] = `
<header>
  <h1>
    Aliens?????
  </h1>
</header>
`;
